pkg/providers/launchtemplate/suite_test.go

```diff
@@ -318,6 +320,109 @@ var _ = Describe("LaunchTemplate Provider", func() {
 			Expect(awsEnv.EC2API.CreateFleetBehavior.SuccessfulCalls()).To(BeNumerically("==", 2))
 
 		})
+		// Testing launchtemplate hash key will produce unique hashes
+		It("should generate different launchtemplate names based on amifamily option configuration", func() {
```

Comment:
nit: launch template should be two words (applies everywhere)
```suggestion
		It("should generate different launch template names based on amifamily option configuration", func() {
```